packages/hash/frontend/src/blocks/page/Comments/CreateBlockComment.tsx

```diff
@@ -0,0 +1,72 @@
+import { forwardRef, useState } from "react";
+import { Box } from "@mui/material";
+import { IconButton, FontAwesomeIcon } from "@hashintel/hash-design-system";
+import { faComment } from "@fortawesome/free-regular-svg-icons";
+import { TextToken } from "@hashintel/hash-shared/graphql/types";
+import { CommentTextField, CommentTextFieldRef } from "./CommentTextField";
+import styles from "./style.module.css";
+import { useRoutePageInfo } from "../../../shared/routing";
+import { useCreateComment } from "../../../components/hooks/useCreateComment";
+
+type CreateBlockCommentProps = {
+  blockId: string | null;
+  onClose?: () => void;
+};
+
+export const CreateBlockComment = forwardRef<
+  CommentTextFieldRef,
+  CreateBlockCommentProps
+>(({ blockId, onClose }, ref) => {
+  const { pageEntityId } = useRoutePageInfo();
+  const [createComment, { loading }] = useCreateComment(pageEntityId);
+  const [inputValue, setInputValue] = useState<TextToken[]>([]);
+
+  const submitComment = async () => {
```

Comment:
Should this be called `handleCommentSubmit`?

Reply:
Renamed in adc9ca84d0109a6bd8038b449ef4c7bacb0b0b65.